templates/android/template/instantapp/build.gradle

```diff
@@ -71,6 +71,14 @@ android {
             if (project.hasProperty("RELEASE_STORE_FILE")) {
                 signingConfig signingConfigs.release
             }
+
+            externalNativeBuild {
+                cmake {
+                    // switch HIDE_SYMBOLS to OFF to skip compilation flag `-fvisibility=hidden`
+                    arguments "-DHIDE_SYMBOLS=ON"
```

Comment:
Could we define it in template's CMakeLists.txt?

Reply:
Yes, we can override the value of `HIDE_SYMBOLS` in CMakeLists.txt file or cfg.cmake